ui/client/components/AddProcessDialog.tsx

```diff
@@ -85,7 +86,7 @@ class AddProcessDialog extends React.Component<Props, State> {
                     <div className="node-row">
                       <div className="node-label">Name</div>
                       <div className="node-value">
-                        <input
+                        <InputWithFocus
```

Comment:
Do we have Input/Select/ etc WithoutFocus? Can we consider just Input or NussknackerInput? I assume that in future we want extract base componnents to secondary liby.

Reply:
those components are temporarily needed to separate styles